01-JS-DOM-APIs/js/table.js

```diff
@@ -0,0 +1,23 @@
+// arr must be bidimensional
+function makeTable(arr) {
+  var tableNode = document.createElement("table");
+  document.getElementById("table-div").appendChild(tableNode);
+
+  for (var r = 0; r < arr.length; r++) {
+    var rowNode = document.createElement("tr");
```

Comment:
It is better to declare variables outside the for loops in order to avoid declarate this in every loop